data/mods/Magiclysm/effects/effects.json

```diff
@@ -205,11 +205,11 @@
     "id": "eff_frightful_presence",
     "name": [ "Scared", "Frightened", "Terrified" ],
     "max_intensity": 3,
+    "max_duration": "10 s",
     "desc": [ "Your knees are shaking, your heart beats fast, and your stomach rebels." ],
     "apply_message": "You are afraid!",
     "remove_message": "Your fear dissipates.",
-    "rating": "bad",
-    "base_mods": { "stim_amount": [ 12, 20, 45 ], "vomit_chance": [ 2, 5, 20 ], "speed_mod": [ -2, -5, -10 ] }
```

Comment:
why remove this?

Reply:
Summary fixed!

I tried removing just vomit_chance and speed_mod... Resulted in the stim amount for my character going up and up and up each time I left the fear aura and reentered the fear aura, quickly resulting in stim overdose!

That was possibly what was occurring with the fear aura, stacking the speed_mod each time the you moved out and into the aura, but... Even with "max_duration": "10 s" in there, the last effect kept on stacking and stacking without any end in sight.